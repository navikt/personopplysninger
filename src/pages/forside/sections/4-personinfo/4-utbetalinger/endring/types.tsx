import { OptionType } from '../../../../../../types/option';

export interface OutboundNorskKontonummer {
    value: string;
}

export interface OutboundUtenlandsbankonto {
    value: string | null;
    utenlandskKontoInformasjon: {
        landkode: string;
        landkodeTobokstavs?: string;
        valuta: string;
        swift?: string;
        bank: {
            adresseLinje1?: string;
            adresseLinje2?: string;
            adresseLinje3?: string;
            kode?: string;
            navn: string;
        };
    };
}

export interface FormFields {
    kontonummer: string;
    land?: OptionType;
    valuta?: OptionType;
    banknavn: string;
    kontonummerIban: string;
    bickode: string;
    retningsnummer: string;
    bankkode: string;
    adresse1: string;
    adresse2: string;
    adresse3: string;
}
