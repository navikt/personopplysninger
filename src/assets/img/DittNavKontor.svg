<?xml version="1.0" encoding="UTF-8"?>
<svg width="64px" height="64px" viewBox="0 0 64 64" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 55 (78076) - https://sketchapp.com -->
    <title>Group 16@1x</title>
    <desc>Created with Sketch.</desc>
    <defs>
        <rect id="path-1" x="0" y="0" width="64" height="64"></rect>
        <circle id="path-3" cx="32" cy="32" r="32"></circle>
    </defs>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <g id="GDPR-Personopplysning" transform="translate(-597.000000, -3652.000000)">
            <g id="Group-16" transform="translate(597.000000, 3652.000000)">
                <g id="Group-12">
                    <g id="spotlight/-gjenstand-">
                        <mask id="mask-2" fill="white">
                            <use xlink:href="#path-1"></use>
                        </mask>
                        <g id="Background"></g>
                        <g id="spotlight/-element" mask="url(#mask-2)" fill-rule="evenodd" stroke-width="1">
                            <g id="ðŸŒˆ-60">
                                <mask id="mask-4" fill="white">
                                    <use xlink:href="#path-3"></use>
                                </mask>
                                <g id="Mask"></g>
                                <g id="NAV-Brand/farge/-BlÃ¥/-60" mask="url(#mask-4)" fill="#99C2E8">
                                    <polygon id="blÃ¥-60" points="0 64 64 64 64 0 0 0"></polygon>
                                </g>
                            </g>
                        </g>
                    </g>
                </g>
                <g id="Group-22" transform="translate(17.000000, 16.000000)" fill-rule="nonzero">
                    <polygon id="Path" fill="#59514B" points="0 32 9.12353407 9.84615385 22.1538462 22.8764659"></polygon>
                    <polygon id="Path" fill="#337C9B" points="22.8764659 22.1538462 9.84615385 9.12158809 32 0"></polygon>
                </g>
            </g>
        </g>
    </g>
</svg>