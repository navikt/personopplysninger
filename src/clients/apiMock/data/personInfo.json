{
  "personalia": {
    "fornavn": "DONALD FAUNTLEROY",
    "etternavn": "DUCK",
    "personident": {
      "verdi": "12345678911",
      "type": "FNR"
    },
    "utenlandskbank": {
      "adresse1": "Gävlegatan 22",
      "adresse2": "113 30 Stockholm",
      "adresse3": "",
      "bankkode": "90081210188",
      "banknavn": "Nordax Bank AB",
      "iban": "",
      "kontonummer": "US90081210188",
      "land": "Sverige",
      "swiftkode": "HELGSE21",
      "valuta": "Tugriks Mongolia"
    },
    "tlfnr": {
      "jobb": "12121212",
      "mobil": "24242424",
      "privat": "36363636",
      "datakilder": [{}, {}]
    },
    "spraak": "NB",
    "epostadr": "TODO",
    "personstatus": "BOSA",
    "statsborgerskap": "NOR",
    "foedested": "NOR",
    "sivilstand": "UGIF",
    "kjoenn": "M",
    "datakilder": [{}, {}, {}, {}]
  },
  "enhetKontaktInformasjon": {
    "enhet": {
      "postadresse": {
        "type": "stedsadresse",
        "postnummer": "0614",
        "poststed": "Oslo",
        "gatenavn": "Slottsveien",
        "husnummer": "22",
        "husbokstav": "A"
      },
      "publikumsmottak": [
        {
          "gateadresse": "Peder Falcks veg",
          "poststed": "Trondheim",
          "husnummer": "31",
          "husbokstav": null,
          "postnummer": "7044",
          "stedsbeskrivelse": null,
          "aapningMandag": {
            "dag": "Mandag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": "Timeavtaler også utenom åpningstiden"
          },
          "aapningTirsdag": {
            "dag": "Tirsdag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningOnsdag": {
            "dag": "Onsdag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningTorsdag": {
            "dag": "Torsdag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningFredag": {
            "dag": "Fredag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningAndre": [
            {
              "dag": "15.05.2019",
              "fra": "09:00",
              "til": "12:00",
              "stengt": "false",
              "kommentar": "Her kan det også være en merknad "
            },
            {
              "dag": "17.05.2019",
              "fra": null,
              "til": null,
              "stengt": "true"
            },
            {
              "dag": "29.05.2019",
              "fra": "09:00",
              "til": "13:30",
              "stengt": "false",
              "kommentar": "Onsdag før Kristi himmelfartsdag"
            }
          ]
        },
        {
          "gateadresse": "Kirkegata",
          "poststed": "Trondheim",
          "husnummer": "25",
          "husbokstav": null,
          "postnummer": "7014",
          "stedsbeskrivelse": null,
          "aapningMandag": {
            "dag": "Mandag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": "Timeavtaler også utenom åpningstiden"
          },
          "aapningTirsdag": {
            "dag": "Tirsdag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningOnsdag": {
            "dag": "Onsdag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningTorsdag": {
            "dag": "Torsdag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningFredag": {
            "dag": "Fredag",
            "fra": "10:00",
            "til": "14:00",
            "stengt": "false",
            "kommentar": null
          },
          "aapningAndre": [
            {
              "dag": "15.05.2019",
              "fra": "09:00",
              "til": "12:00",
              "stengt": "false",
              "kommentar": "Her kan det også være en merknad "
            },
            {
              "dag": "17.05.2019",
              "fra": null,
              "til": null,
              "stengt": "true"
            },
            {
              "dag": "29.05.2019",
              "fra": "09:00",
              "til": "13:30",
              "stengt": "false",
              "kommentar": "Onsdag før Kristi himmelfartsdag"
            }
          ]
        }
      ],
      "tlfperson": "55553333",
      "spesielleopplysninger": "Timeavtale 8-15, man - fre."
    }
  },
  "adresser": {
    "boadresse": {
      "adressetillegg": "C/O Rune Aremark",
      "adresse": "Bjørvika 4D",
      "datoFraOgMed": "2017-02-21",
      "kommune": "1239",
      "land": "Norge",
      "matrikkeladresse": {},
      "postnummer": "1239",
      "poststed": "Jepp",
      "veiadresse": {
        "bokstav": "A",
        "bolignummer": "H0102",
        "gatekode": "01239",
        "husnummer": "11"
      }
    },
    "geografiskTilknytning": {
      "bydel": "500102",
      "datoFraOgMed": "2012-10-01",
      "kommune": null,
      "land": null,
      "enhet": "NAV Falkenborg"
    },
    "postadresse": {
      "adresse1": "12345 HFJEF IQUFJ HFURHGHQBEUQ",
      "adresse2": "BATOOM LUNCHAGRAMA,GONGRUPEA,DD",
      "adresse3": "GGLAM,10000 HCAMGHU,THAILAND",
      "datoFraOgMed": "2018-03-13",
      "land": "THA",
      "postnummer": "",
      "poststed": "Nittedal"
    },
    "prioritertAdresse": "TIAD",
    "tilleggsadresse": {
      "adresse1": "JEPP 10jfioew JIO LPFL ",
      "adresse2": "LOOOL FKEOWIF JEWIOF WECOOOO FJEI",
      "adresse3": "",
      "bolignummer": "",
      "bydel": "",
      "datoFraOgMed": "2019-01-12",
      "datoTilOgMed": "2023-01-11",
      "gateKode": "0270",
      "husbokstav": "",
      "husnummer": "0020",
      "kommunenummer": "0131",
      "postboksanlegg": "",
      "postboksnummer": "",
      "postnummer": "2021",
      "poststed": "JAJAJA"
    },
    "datakilder": [{}, {}]
  }
}
