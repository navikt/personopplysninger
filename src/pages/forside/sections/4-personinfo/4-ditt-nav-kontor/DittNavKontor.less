.dittnavkontor {
  &__header {
    margin: 1rem 0 2rem 0;
  }
  &__ingress {
    display: flex;
    margin-bottom: 1rem;

    & > * {
      margin-right: 0.25rem !important;
    }
  }

  &__footer {
    margin-top: 1rem !important;
  }
  &__adresser {
    display: flex;
    flex-wrap: wrap;
  }
  &__publikumsmottak,
  &__postadresse {
    width: 50%;
    @media @maxMobile {
      padding-bottom: 1rem;
      width: 100%;
    }
  }
}
