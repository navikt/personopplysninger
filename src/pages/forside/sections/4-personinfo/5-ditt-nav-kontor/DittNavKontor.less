.dittnavkontor {
  &__header {
    @media @minMobile {
      margin: 0 0 2rem 0;
    }
    @media @maxMobile {
      margin: 0 0 1rem 0;
    }
  }
  &__ingress {
    display: flex;
    margin-bottom: 1rem;

    & > * {
      margin-right: 0.25rem !important;
    }
  }

  &__footer {
    margin-top: 1rem !important;
  }
  &__adresser {
    display: flex;
    flex-wrap: wrap;
    margin-top: 1rem;
  }
  &__publikumsmottak,
  &__postadresse {
    width: 50%;
    @media @maxMobile {
      padding-bottom: 1rem;
      width: 100%;
    }
  }
}

.selectContainer {
  margin-bottom: 1rem;
}
