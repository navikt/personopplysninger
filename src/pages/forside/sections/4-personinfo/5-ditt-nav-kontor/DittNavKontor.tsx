import Box from 'components/box/Box';
import dittNavKontorIkon from 'assets/img/DittNavKontor.svg';
import { EnhetKontaktInfo } from 'types/enhetKontaktInfo';
import { GeografiskTilknytning } from 'types/adresser';
import { FormattedMessage } from 'react-intl';
import ListElement from 'components/listelement/ListElement';
import Kilde from 'components/kilde/Kilde';
import { useStore } from 'store/Context';
import { BodyShort, Label, Link } from '@navikt/ds-react';
import { Reception } from '@navikt/nav-office-reception-info';

import '@navikt/nav-office-reception-info/dist/style.css';

interface Props {
    enhetKontaktInformasjon: EnhetKontaktInfo;
    geografiskTilknytning?: GeografiskTilknytning;
}

const DittNavKontor = (props: Props) => {
    const [{ locale }] = useStore();

    const enhetNavn = props.enhetKontaktInformasjon.navn;
    const publikumsmottak = props.enhetKontaktInformasjon.brukerkontakt.publikumsmottak;

    if (!enhetNavn || !publikumsmottak) {
        return null;
    }

    return (
        <Box id="ditt-nav-kontor" tittel="dittnavkontor.tittel" beskrivelse="dittnavkontor.beskrivelse" icon={dittNavKontorIkon} visAnkerlenke>
            <div className="dittnavkontor__ingress">
                <BodyShort>
                    <FormattedMessage id="dittnavkontor.ingress" />
                    <Label as="span">{enhetNavn}</Label>
                </BodyShort>
            </div>
            <Reception receptions={publikumsmottak} language={locale} />
            <dl className="dittnavkontor__footer list">
                <ListElement
                    titleId="dittnavkontor.kontaktinfo.overskrift"
                    content={
                        <Link href={`/person/kontakt-oss/${locale === 'en' ? 'en' : 'nb'}/`}>
                            <FormattedMessage id="dittnavkontor.kontaktinfo.lenke" />
                        </Link>
                    }
                />
            </dl>
            <Kilde kilde="personalia.source.nav" lenkeType={'INGEN'} />
        </Box>
    );
};

export default DittNavKontor;
