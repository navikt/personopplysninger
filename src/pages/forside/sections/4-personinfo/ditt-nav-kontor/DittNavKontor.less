.dittnavkontor {
  &__header {
    margin: 1rem 0 2rem 0;
  }
  &__ingress {
    display: flex;
    margin-bottom: 1rem;

    & > * {
      margin-right: 0.25rem !important;
    }
  }

  &__footer {
    margin-top: 2rem;
  }
}
