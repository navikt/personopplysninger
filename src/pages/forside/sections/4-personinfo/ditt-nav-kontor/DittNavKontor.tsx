import React, { ChangeEvent, useState } from "react";
import Box from "../../../../../components/box/Box";
import dittNavKontorIkon from "../../../../../assets/img/DittNavKontor.svg";
import { Enhet } from "../../../../../types/enhetKontaktInfo";
import { GeografiskTilknytning } from "../../../../../types/adresser";
import { Normaltekst, Element } from "nav-frontend-typografi";
import { FormattedMessage, FormattedHTMLMessage } from "react-intl";
import { Select } from "nav-frontend-skjema";
import { injectIntl, InjectedIntlProps } from "react-intl";
import ListElement from "../../../../../components/listelement/ListElement";
import Apningstid from "../../../../../components/apningstid/Apningstid";
import { print } from "../../../../../utils/text";
import Kilde from "../../../../../components/kilde/Kilde";

interface Props {
  enhetKontaktInfo: Enhet;
  geografiskTilknytning: GeografiskTilknytning;
}

const DittNavKontor = (props: Props & InjectedIntlProps) => {
  const { enhetKontaktInfo, geografiskTilknytning, intl } = props;
  const { publikumsmottak, postadresse } = enhetKontaktInfo;
  const [valgtMottakId, settValgtMottakId] = useState(
    publikumsmottak.length > 1 ? -1 : 0
  );

  return (
    <Box
      id="dittnavkontor"
      tittel="dittnavkontor.tittel"
      beskrivelse="dittnavkontor.beskrivelse"
      icon={dittNavKontorIkon}
    >
      <hr className="box__linje-bred" />
      <div className="dittnavkontor__header">
        <div className="dittnavkontor__ingress">
          <Normaltekst>
            <FormattedMessage id="dittnavkontor.ingress" />
          </Normaltekst>
          <Element>{geografiskTilknytning.enhet}</Element>
        </div>
        {publikumsmottak.length > 1 && (
          <Select
            label=""
            onChange={(event: ChangeEvent<HTMLSelectElement>) => {
              settValgtMottakId(parseInt(event.currentTarget.value));
            }}
          >
            <option value="-1">
              {`${intl.formatMessage({
                id: "dittnavkontor.publikumsmottakfor"
              })} ${geografiskTilknytning.enhet}`}
            </option>
            {publikumsmottak.map((mottak, id) => (
              <option key={id} value={id}>
                {`${print(mottak.poststed).toUpperCase()} - ${print(
                  mottak.gateadresse
                )} ${print(mottak.husnummer)}${print(
                  mottak.husbokstav
                )}, ${print(mottak.postnummer)} ${print(mottak.poststed)}`}
              </option>
            ))}
          </Select>
        )}
        <div className="dittnavkontor__adresser">
          {postadresse && (
            <div className="dittnavkontor__postadresse">
              <Element>
                <FormattedMessage id="dittnavkontor.postadresse" />
              </Element>
              <Normaltekst>
                {postadresse.type === "stedsadresse" &&
                  `${print(postadresse.gatenavn)} ${print(
                    postadresse.husnummer
                  )} ${print(postadresse.husbokstav)}`}
                {postadresse.type === "postboksadresse" &&
                  `${props.intl.formatMessage({
                    id: "dittnavkontor.postboks"
                  })} ${print(postadresse.postboksnummer)} ${print(
                    postadresse.postboksanlegg
                  )}`}
              </Normaltekst>
              <Normaltekst>
                {print(postadresse.postnummer)} {print(postadresse.poststed)}
              </Normaltekst>
            </div>
          )}
          {valgtMottakId !== -1 && (
            <div className="dittnavkontor__publikumsmottak">
              <Element>
                <FormattedMessage id="dittnavkontor.publikumsmottak" />
              </Element>
              <Normaltekst>
                {`${print(publikumsmottak[valgtMottakId].gateadresse)} ${print(
                  publikumsmottak[valgtMottakId].husnummer
                )}${print(publikumsmottak[valgtMottakId].husbokstav)}`}
              </Normaltekst>
              <Normaltekst>
                {`${print(publikumsmottak[valgtMottakId].postnummer)} ${print(
                  publikumsmottak[valgtMottakId].poststed
                )}`}
              </Normaltekst>
            </div>
          )}
        </div>
      </div>
      <div>
        {valgtMottakId !== -1 ? (
          <>
            <Element>
              <FormattedMessage id="dittnavkontor.apningstider" />
            </Element>
            <div className="apningstid__container">
              <Apningstid
                apningstid={publikumsmottak[valgtMottakId].aapningMandag}
              />
              <Apningstid
                apningstid={publikumsmottak[valgtMottakId].aapningTirsdag}
              />
              <Apningstid
                apningstid={publikumsmottak[valgtMottakId].aapningOnsdag}
              />
              <Apningstid
                apningstid={publikumsmottak[valgtMottakId].aapningTorsdag}
              />
              <Apningstid
                apningstid={publikumsmottak[valgtMottakId].aapningFredag}
              />
            </div>
            {publikumsmottak[valgtMottakId].aapningAndre && (
              <>
                <Element>
                  <FormattedMessage id="dittnavkontor.andreapningstider" />
                </Element>
                <div className="apningstid__container">
                  {publikumsmottak[valgtMottakId].aapningAndre!.map(
                    (apningstid, id) => (
                      <Apningstid key={id} apningstid={apningstid} />
                    )
                  )}
                </div>
              </>
            )}
          </>
        ) : (
          <hr />
        )}
      </div>
      <ul className="dittnavkontor__footer list-column-2">
        <ListElement
          titleId="dittnavkontor.kontaktinfo.kontaktsenter.tittel"
          content={
            <FormattedHTMLMessage id="dittnavkontor.kontaktinfo.kontaktsenter.tlfnr" />
          }
        >
          <>
            <FormattedHTMLMessage id="dittnavkontor.kontaktinfo.pensjon.tlfnr" />
            (<FormattedMessage id="dittnavkontor.kontaktinfo.pensjon" />)
          </>
        </ListElement>
        <ListElement
          titleId="dittnavkontor.kontaktinfo.apningstider.tittel"
          content={intl.formatMessage({
            id: "dittnavkontor.kontaktinfo.apningstider.innhold"
          })}
        />
      </ul>
      <Kilde kilde="personalia.source.nav" />
    </Box>
  );
};

export default injectIntl(DittNavKontor);
