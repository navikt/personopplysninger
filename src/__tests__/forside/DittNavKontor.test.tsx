import '@testing-library/jest-dom';
import { IntlProvider } from 'react-intl';
import nbMessages from 'text/nb';
import personInfo from '../../clients/apiMock/app/fetch/person-info.json';
import { StoreProvider } from '../../store/Context';
import DittNavKontor from '../../pages/forside/sections/4-personinfo/5-ditt-nav-kontor/DittNavKontor';
import { EnhetKontaktInfo } from '../../types/enhetKontaktInfo';
import { render } from '@testing-library/react';

jest.mock('react-modal');

//Mock import som feiler
jest.mock('@navikt/nav-office-reception-info', () => ({
    Reception: () => <div>Mock Reception</div>,
}));

describe('DittNavKontor', () => {
    it.skip('renders correctly', () => {
        const { asFragment } = render(
            <StoreProvider>
                <IntlProvider locale={'nb'} messages={nbMessages}>
                    <DittNavKontor enhetKontaktInformasjon={personInfo.enhetKontaktInformasjon as unknown as EnhetKontaktInfo} />
                </IntlProvider>
            </StoreProvider>
        );
        expect(asFragment()).toMatchSnapshot();
    });
});
