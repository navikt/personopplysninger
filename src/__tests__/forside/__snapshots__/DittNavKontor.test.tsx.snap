// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DittNavKontor renders correctly 1`] = `
<DocumentFragment>
  <div
    class="box__wrapper"
    id="ditt-nav-kontor"
  >
    <div
      class="navds-guide-panel navds-guide-panel--poster"
    >
      <div
        class="navds-guide"
      >
        <div
          class="navds-guide__illustration navds-guide__illustration--medium"
        >
          <img
            alt=""
            class="box__ikon"
            src="DittNavKontor.svg"
          />
        </div>
      </div>
      <div
        class="navds-guide-panel__content"
      >
        <div
          class="box__container"
        >
          <div
            class="box__header"
          >
            <div
              class="box__title-container"
            >
              <div
                class="box__line"
              />
              <h2
                class="box__title navds-heading navds-heading--medium"
              >
                Ditt NAV-kontor
              </h2>
              <button
                aria-label="Les mer om denne seksjonen."
                aria-pressed="false"
                class="infotekst__title-i-button"
              >
                <img
                  alt=""
                  class="infotekst__title-i-icon"
                  src="Informasjonstekst.svg"
                />
              </button>
              <div
                class="box__line"
              />
            </div>
            <a
              class="navds-link anchor-link"
              href="#ditt-nav-kontor"
            >
              <img
                alt=""
                src="AnchorLink.svg"
              />
              Lenke hit
            </a>
          </div>
          <div
            class="box__content"
          >
            <div
              class="dittnavkontor__header"
            >
              <div
                class="dittnavkontor__ingress"
              >
                <p
                  class="navds-body-short"
                >
                  Ditt kontor er 
                  <span
                    class="navds-label"
                  >
                    NAV Halden-Aremark
                  </span>
                </p>
              </div>
              <div
                class="navds-form-field navds-form-field--medium"
              >
                <label
                  class="navds-form-field__label navds-sr-only navds-label"
                  for="textField-r1"
                >
                  NAV-kontor
                </label>
                <div
                  class="navds-select__container"
                >
                  <select
                    aria-invalid="false"
                    class="navds-select__input navds-body-short navds-body--medium"
                    id="textField-r1"
                    name="NAV-kontor"
                  >
                    <option
                      value="-1"
                    >
                      Publikumsmottak for NAV Halden-Aremark
                    </option>
                    <option
                      value="0"
                    >
                      HALDEN - Storgata 8, 1771 HALDEN
                    </option>
                    <option
                      value="1"
                    >
                      AREMARK - Aremark Rådhus , 1798 AREMARK
                    </option>
                  </select>
                  <svg
                    aria-hidden="true"
                    class="navds-select__chevron"
                    fill="none"
                    focusable="false"
                    height="1em"
                    role="img"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      clip-rule="evenodd"
                      d="M12 17.414 4.293 9.707l1.414-1.414L12 14.586l6.293-6.293 1.414 1.414L12 17.414Z"
                      fill="currentColor"
                      fill-rule="evenodd"
                    />
                  </svg>
                </div>
                <div
                  aria-live="polite"
                  aria-relevant="additions removals"
                  class="navds-form-field__error"
                  id="textField-error-r1"
                />
              </div>
              <div
                class="dittnavkontor__adresser"
              >
                <div
                  class="dittnavkontor__postadresse"
                >
                  <h3
                    class="navds-heading navds-heading--xsmall"
                  >
                    Postadresse
                  </h3>
                  <p
                    class="navds-body-short"
                  >
                    Postboks 56 
                  </p>
                  <p
                    class="navds-body-short"
                  >
                    1751 HALDEN
                  </p>
                </div>
                <div
                  class="dittnavkontor__publikumsmottak"
                >
                  <h3
                    class="navds-heading navds-heading--xsmall"
                  >
                    Publikumsmottak
                  </h3>
                  <p
                    class="navds-body-short"
                  >
                    Storgata 8
                  </p>
                  <p
                    class="navds-body-short"
                  >
                    1771 HALDEN
                  </p>
                </div>
              </div>
            </div>
            <div>
              <div
                class="apningstider"
              >
                <table
                  class="apningstider__table"
                >
                  <caption
                    class="apningstider__caption navds-heading navds-heading--xsmall"
                  >
                    Åpningstider
                  </caption>
                  <thead
                    class="sr-only"
                  >
                    <tr>
                      <th
                        scope="col"
                      >
                        Ukedag
                      </th>
                      <th
                        scope="col"
                      >
                        Tidsrom
                      </th>
                      <th
                        scope="col"
                      >
                        Kommentar
                      </th>
                    </tr>
                  </thead>
                  <tbody>
                    <tr>
                      <td>
                        Mandag
                      </td>
                      <td>
                        12:30 - 14:30
                      </td>
                      <td />
                    </tr>
                    <tr>
                      <td>
                        Tirsdag
                      </td>
                      <td>
                        12:30 - 14:30
                      </td>
                      <td />
                    </tr>
                    <tr>
                      <td>
                        Onsdag
                      </td>
                      <td>
                        12:30 - 14:30
                      </td>
                      <td />
                    </tr>
                    <tr>
                      <td>
                        Torsdag
                      </td>
                      <td>
                        12:30 - 14:30
                      </td>
                      <td />
                    </tr>
                    <tr>
                      <td>
                        Fredag
                      </td>
                      <td>
                        12:30 - 14:30
                      </td>
                      <td />
                    </tr>
                  </tbody>
                </table>
                <table
                  class="apningstider__table"
                >
                  <caption
                    class="apningstider__caption navds-heading navds-heading--xsmall"
                  >
                    Spesielle åpningstider
                  </caption>
                  <thead
                    class="sr-only"
                  >
                    <tr>
                      <th
                        scope="col"
                      >
                        Dato
                      </th>
                      <th
                        scope="col"
                      >
                        Tidsrom
                      </th>
                      <th
                        scope="col"
                      >
                        Kommentar
                      </th>
                    </tr>
                  </thead>
                  <tbody>
                    <tr>
                      <td>
                        24.12.2021
                      </td>
                      <td>
                        10:00 - 11:30
                      </td>
                      <td>
                        Åpent for akutte behov for alle NAV-kontorer i Bergen kommune
                      </td>
                    </tr>
                    <tr>
                      <td>
                        31.12.2021
                      </td>
                      <td>
                        Stengt
                      </td>
                      <td>
                        NAV Bergen vest er åpent 10-11.30 for akutte behov
                      </td>
                    </tr>
                  </tbody>
                </table>
              </div>
            </div>
            <dl
              class="dittnavkontor__footer list"
            >
              <div
                class="list-item"
              >
                <dt
                  class="list-element-heading"
                >
                  <span
                    class="navds-label"
                  >
                    Kontakt NAV
                  </span>
                </dt>
                <dd>
                  <a
                    class="navds-link"
                    href="/person/kontakt-oss/nb/"
                  >
                    Gå til kontaktinformasjon
                  </a>
                </dd>
              </div>
            </dl>
            <div
              class="kilde__container"
            >
              <div
                class="kilde__tekst"
              >
                <p
                  class="navds-detail"
                >
                  Kilde: NAV
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</DocumentFragment>
`;
